lib/src/scss-shared/_nx-tag-variables.scss

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../scss-shared/nx-dark-mode-helpers';
+
+.nx-selectable-color--light-blue, // deprecated
+.nx-selectable-color--sky {
+  --nx-tag-selected-hover-border: var(--nx-color-interactive-border-hover);
+  --nx-tag-selected-hover-background: var(--nx-selectable-color-dark);
+  --nx-tag-unselected-hover-border: var(--nx-color-interactive-border-hover);
+  --nx-tag-unselected-hover-background: var(--nx-selectable-color-light);
```

Comment:
The light mode uses the same values for each one? There are probably ways to cut out that repetition, either by putting that into a `:root` block or into a placeholder selector that each of these blocks `@extends`.

Reply:
> The light mode uses the same values for each one? There are probably ways to cut out that repetition, either by putting that into a :root block or into a placeholder selector that each of these blocks @extends.

Looks like the dark mode hover borders are also always the same.  So lets move all of the common ones into a `:root` block.  Maybe even back into `_nx-colors.scss`.

Reply:
> > The light mode uses the same values for each one? There are probably ways to cut out that repetition, either by putting that into a :root block or into a placeholder selector that each of these blocks @extends.
> 
> Looks like the dark mode hover borders are also always the same. So lets move all of the common ones into a `:root` block. Maybe even back into `_nx-colors.scss`.

Went with this approach: 9f8ead04a361a509d5708121b3c0d310de113a37